File: mario.c
#include <cs50.h>
#include <stdio.h>



int main(void)
{
    int Height; //Declare integer tag for height
    
    //////////// Require and check Height between 1 and 9/////////
    do
    {
    Height = get_int("Height: ");    
    } 
    while (Height <= 0 || Height > 8);
    //////////////////////////////////////////////////
        
     for (int i = 0; i < Height; i++)  ///// loop for the levels of pyramid
     {
         for (int j = 0; j < (Height-i-1); j++)   //// loop for spaces of the left side
         {
          printf (" "); 
         } 
                  
         for (int k = 0; k <= i; k++)   /// loop for left side of the pyramid (blocks)
         {
          printf ("#"); 
         } 
         
          printf ("  ");  ////// central spaces
         
           for (int l = 0; l <= i; l++)   ///// loop for right side of the pyramid (blocks)
         {
          printf ("#");   
         }
         
          printf ("\n");  /// final jump   
      }    
    
}

File: mario.py
text = 0
number = 0

#check for a valid input
while number < 1 or number > 8:

    text = input("Height: ")
    #if is a decimal convert str to int
    if text.isdecimal() == True:
        number = int(text)
    #if is an alphabetic, set 999
    elif text.isalpha() == True:
        number = 999
    #if something else, set 777
    else:
        number = 777

#print every level
for i in range(number):
    space = number - i - 1

    #print left side spaces
    for j in range(space):
        print(" ", end = '')

    #print left side #
    for k  in range(i + 1):
        print("#", end = ''),

    #print two central spaces
    print("  ", end = '')

    #print right side #
    for l  in range(i + 1):
        print("#", end = ''),

    #print right side spaces
    for m in range(space):
        print(" ", end = ''),
    #finish print actual level and jump to the next one
    else:
        print("")
